frontend/src/app/trees/forests/forest-finder/forest-finder.component.html

```diff
@@ -1,5 +1,8 @@
 <section class="usa-section">
   <div class="usa-grid">
+    <div class="breadcrumbs">
+      <a routerLink="/forests">Christmas tree permits</a> > <span>Find a forest</span>
```

Comment:
I'm confused by the use of a naked `>` in HTML. Is this processed in some fashion to convert it to `&gt;`?